web-common/src/runtime-client/RuntimeProvider.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import { runtime } from "./runtime-store";
+
+  export let host: string;
+  export let instanceId: string;
+  export let jwt: string = undefined;
+
+  $: runtime.set({
+    host: host,
+    instanceId: instanceId,
+    jwt: jwt,
+  });
+</script>
+
+<slot />
```

Comment:
1. If the host/instanceID/JWT is initially unset (only populated after project lookup resolves), will it prevent the dashboard from rendering (instead of showing errors)
2. If the runtime host/instanceID are changed, will the full dashboard reload?

Reply:
Good calls. Just added a couple commits to address.